.circleci/config.yml

```diff
@@ -203,6 +220,21 @@ jobs:
 
 
 workflows:
+  compatibility_checks:
+    jobs:
+      - check_compatibility:
+          python_version: "3.8"
+          name: check-compatibility-3.8
+      - check_compatibility:
+          python_version: "3.9"
+          name: check-compatibility-3.9
+      - check_compatibility:
+          python_version: "3.10"
+          name: check-compatibility-3.10
+      - check_compatibility:
+          python_version: "3.11"
+          name: check-compatibility-3.11
```

Comment:
Do we need this for every commit ?
Maybe we can trigger this just when requirements changes?

Reply:
This is a good point. Probably overkill. All good 8114e1018d4364180ab8514da61f53d6a2ec8f17